packages/ui/components/ui/block-sync-status/large.tsx

```diff
@@ -104,7 +104,7 @@ const FullySyncedState = ({ latestKnownBlockHeight, fullSyncHeight }: SyncingSta
             width='50'
             color='var(--teal)'
             wrapperClass={cn(
-              'transition-all duration-300 absolute bottom-0 right-3',
+              'transition-all duration-300 absolute bottom-0 -left-7',
```

Comment:
This guy has been annoying me as well. This way, as the number grows, it will always be spaced out.

<img width="83" alt="Screenshot 2024-03-18 at 9 16 06 PM" src="https://github.com/penumbra-zone/web/assets/16624263/97c229de-f14d-4fc1-8418-cfa07badb52e">